atmospheric_explorer/plotting_apis.py

```diff
@@ -327,9 +329,12 @@ def eac4_anomalies_plot(
     df_down = xr.open_dataset(data.file_full_path)
     df_down = shifting_long(df_down)
     df_down = df_down.rio.write_crs("EPSG:4326")
-    df_clipped = clip_and_concat_countries(df_down, countries).sel(admin=countries[0])
+    if shapes is not None:
```

Comment:
Am I understanding correctly that the logic in both this and the GHG functions is the same? That is, if `shapes is None`, we are keeping the entire dataset without clipping? 
I'm asking because they seem to be implemented differently: we have an `else` statement for GHG and not here. If the logic is the same, why are we doing it differently?

Reply:
The `else` statement in ghg is because it's the only plot with the facet functionality, so it needs a col num anyway. I simplified it

Reply:
But my question is: does it make sense that, if we have no `shapes`, we are still creating the Anomalies plot? 
It probably does, if we also want to have the possibility to create this kind of plot for the whole world/dataset. I think we should just make it a bit more explicit (or keep it in mind for the documentation).

Reply:
Yes you're right, we should specify it. Otherwise, we can clip this to the selected (i.e. country boxxed) lat and long